<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">

 <class>CalendarWindow</class>
 <widget class="QWidget" name="Planner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planner</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>411</width>
     <height>275</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;
selection-background-color: #0022FF;
    </string>
   </property>
  </widget>
  <widget class="QPushButton" name="todayButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>111</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #0000FF;
color:white;
</string>
   </property>
   <property name="text">
    <string>Today</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasksListWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #0000FF;
color:white;
font:11pt;</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #0000FF;
color:white;
</string>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 24pt;
background : #0000FF;
color:white;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Daily Task Planner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;</string>
   </property>
  </widget>

  <widget class="QMessageBox" name="messageBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:6pt;</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>